execution-engine/engine-tests/src/support/exec_with_return.rs

```diff
@@ -83,11 +83,20 @@ where
         from_account
     };
 
+    let validated_args = arguments
+        .into_iter()
+        .map(|value_bytes| {
+            let (value, _rest) = Value::from_bytes(&value_bytes).unwrap();
```

Comment:
```suggestion
            let value: Value = contract_api::deserialize(&value_bytes);
```

(see above)

Don't unwrap!